src/Console/InfectionContainer.php

```diff
@@ -333,7 +332,8 @@ public static function create(): self
                     $config->getTmpDir(),
                     $container['timer'],
                     $container['time.formatter'],
-                    $container['memory.formatter']
+                    $container['memory.formatter'],
+                    $container[LoggerFactory::class]
```

Comment:
As mentioned somewhere else, I would do the effort to put it in a temporary variable. I think this effort is really necessary for now otherwise IDE inspections, which are by no means mandatory but helps a lot, won't work in cases like those.